var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -49,20 +51,23 @@ class PyFenicsDolfinx(PythonPackage):
     depends_on("fenics-dolfinx@0.1.0", when="@0.1.0")
 
     depends_on("fenics-basix@main", type=("build", "link"), when="@main")
+    depends_on("fenics-basix@0.6.0:0.6", type=("build", "link"), when="@0.6.0:0.6")
     depends_on("fenics-basix@0.5.1:0.5", type=("build", "link"), when="@0.5.0:0.5")
     depends_on("fenics-basix@0.4.2", type=("build", "link"), when="@0.4.1")
     depends_on("fenics-basix@0.3.0", type=("build", "link"), when="@0.3.0")
     depends_on("fenics-basix@0.2.0", type=("build", "link"), when="@0.2.0")
     depends_on("fenics-basix@0.1.0", type=("build", "link"), when="@0.1.0")
 
     depends_on("py-fenics-ffcx@main", type="run", when="@main")
+    depends_on("py-fenics-ffcx@0.6.0:0.6", type="run", when="@0.6.0:0.6")
     depends_on("py-fenics-ffcx@0.5.0.post0", type="run", when="@0.5.0:0.5")
     depends_on("py-fenics-ffcx@0.4.2", type="run", when="@0.4.1")
     depends_on("py-fenics-ffcx@0.3.0", type="run", when="@0.3.0")
     depends_on("py-fenics-ffcx@0.2.0", type="run", when="@0.2.0")
     depends_on("py-fenics-ffcx@0.1.0", type="run", when="@0.1.0")
 
     depends_on("py-fenics-ufl@main", type="run", when="@main")
+    depends_on("py-fenics-ufl@2023.1.1.post0", type="run", when="@0.6.0:0.6")
     depends_on("py-fenics-ufl@2022.2.0", type="run", when="@0.5.0:0.5")
     depends_on("py-fenics-ufl@2022.1.0", type="run", when="@0.4.1")
     depends_on("py-fenics-ufl@2021.1.0", type="run", when="@0.1:0.3.99")
```

Comment:
Where does the lower bound for pybind11 come from?

Reply:
Updated - version requirement in the CMakefile.txt that builds the wrapper interface.